### PR TITLE
Add scenarios for testing Task endpoint

### PR DESCRIPTION
**Pull request by alszla**

### CODE REVIEW

**Review by MelissaRoman06 — APPROVED** [review]

File: src/test/resources/features/task/CreateTask.feature [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    Given user specifies body content
+      | Status   | Not Started |
+      | Priority | Low         |
+    When user posts to Task endpoint
+    Then status code is 201
+    And response includes the following
+      | success | true |
+    And response complies task schema
+
+#  @CreateTasks @Functional
```
Comment (TheYisusGithub) [inline src/test/resources/features/task/CreateTask.feature]: Please remove that comment.

File: src/test/resources/features/task/TaskAcceptance.feature [hunk 2]
```diff
@@ -10,6 +11,22 @@ Feature: Manage Tasks in Salesforce
       | success | true |
     And response complies task schema
 
+#  @CreateTasks
```
Comment (TheYisusGithub) [inline src/test/resources/features/task/TaskAcceptance.feature:14]: Please remove that comment.

File: src/test/resources/features/task/UpdateTask.feature [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+  @Negative
+  Scenario: User updates a task by specifying non-exisiting id and at least a status and priority
+    Given user specifies body content
+      | Status   | Deferred    |
+      | Priority | High        |
+    When user patches Task 00T3i000005aHNREA2
+    Then status code is 404
+
+#  @UpdateTasks @Functional
```
Comment (TheYisusGithub) [inline src/test/resources/features/task/UpdateTask.feature]: Please remove that comment.

File: src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java [hunk 4]
```diff
@@ -107,7 +135,8 @@ public void response_is_valid (String schemaTypeName) {
      */
     @When("user patches an existing task")
     public void user_patches_content() {
-        this.response = taskApi.patchContent(task.getId());
+        this.response = taskApi.patchContent(context.getTask().getId());
+        System.out.println("imprimiendo patches");
```
Comment (aburgosu) [inline src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java]: Delete SOUT

File: src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java [hunk 5]
```diff
@@ -127,14 +156,15 @@ public void user_patches_content(String id) {
      */
     @When("user searches for an existing task")
     public void user_searches_existing(){
-        this.response = taskApi.findTaskById(task.getId());
+        this.response = taskApi.findTaskById(context.getTask().getId());
+        System.out.println("imprimiendo searches");
```
Comment (aburgosu) [inline src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java]: Delete SOUT

File: src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java [hunk 6]
```diff
@@ -145,14 +175,15 @@ public void user_searches_for(String taskId){
      */
     @When("user makes a delete request for an existing task")
     public void user_makes_delete_request_existing(){
-        this.response = taskApi.deleteTaskById(task.getId());
+        this.response = taskApi.deleteTaskById(context.getTask().getId());
+        System.out.println("imprimiendo deletes");
```
Comment (aburgosu) [inline src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java]: Delete SOUT

File: src/test/resources/features/task/CreateTask.feature (hunk 1 above)
Comment (alszla) [reply src/test/resources/features/task/CreateTask.feature]: Done.

File: src/test/resources/features/task/TaskAcceptance.feature (hunk 2 above)
Comment (alszla) [reply src/test/resources/features/task/TaskAcceptance.feature:14]: Done.

File: src/test/resources/features/task/UpdateTask.feature (hunk 3 above)
Comment (alszla) [reply src/test/resources/features/task/UpdateTask.feature]: Done.

File: src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java (hunk 4 above)
Comment (alszla) [reply src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java]: Done.

File: src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java (hunk 5 above)
Comment (alszla) [reply src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java]: Done.

File: src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java (hunk 6 above)
Comment (alszla) [reply src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java]: Done.

**Review by Cxristian — APPROVED** [review]

**Pull request merged**